src/renderer/views/Channel/Channel.js

```diff
@@ -509,6 +542,34 @@ export default Vue.extend({
       })
     },
 
+    getShortsInvidious: function () {
+      const payload = {
+        resource: `channels/${this.id}/shorts`,
+        id: '',
+        params: {
+          sort_by: this.playlistSortBy
+        }
+      }
+      this.invidiousAPICall(payload).then((response) => {
+        this.latestShorts = response.videos
+        this.videoContinuationString = response.continuation
```

Comment:
Should probably add a separate variable for storing the shorts continuation

Reply:
✅